index.html

```diff
@@ -53,6 +48,171 @@ <h2>I am a Software Developer</h2>
+                        <li class="languages">
+                            Languages
+                            <ul>
+                                <li class="skill-item">Javascript</li>
+                                <li class="skill-item">Ruby</li>
+                                <li class="skill-item">Html</li>
+                                <li class="skill-item">Css</li>
+                            </ul>
+                        </li>
+                        <li class="frameworks">
+                            Frameworks
+                            <ul>
+                                <li class="skill-item">Bootstrap</li>
+                                <li class="skill-item">Ruby on Rails</li>
+                                <li class="skill-item">RSPec</li>
+                                <li class="skill-item">CapyBara</li>
+                                <li class="skill-item">Selenium</li>
+                            </ul>
+                        </li>
+                        <li class="skills">
+                            Skills
+                            <ul>
+                                <li class="skill-item">Codekit</li>
+                                <li class="skill-item">Github</li>
+                                <li class="skill-item">Codepen</li>
+                                <li class="skill-item">Gitlab</li>
+                                <li class="skill-item">Terminal</li>
+                            </ul>
+                        </li>
+                    </ul>
+    
+                    <br>
+                </div>
+            </section><!-- #About me -->
+            <!-- ================
+              contact-me
+             ================ -->
+        <section class="contact-me">
+            <form action="https://formspree.io/f/moqrjalz" method="post">
+                <h6>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h6>
+                <input name="fullname" type="text" placeholder="Full name" minlength="5" maxlength="50" required>
+                <input name="email" type="email" placeholder="Email address" required>
+                <textarea name="message" minlength="5" maxlength="500" required>Enter text here</textarea>
+                <button type="submit">Get in touch</button>
+            </form>
+        </section><!-- #contact-me -->
+        <!-- ================
+              Footer
+          ================ -->
+    <footer class="footer">
+        <div class="social-icon">
+            <ul>
+                <li><a href="#"><i class="fab fa-github-alt "></i></a></li>
+                <li><a href="#"><i class="fab fa-linkedin "></i></a></li>
+                <li><a href="#"><i class="fab fa-twitter-square "></i></a></li>
+                <li><a href="#"><i class="fab fa-youtube "></i></a></li>
+                <li><a href="#"><i class="fab fa-medium "></i></a></li>
+            </ul>
```

Comment:
Kindly make the necessary changes to your code so that your social links are properly aligned. As you can imagine, the issue has to do with the one I mentioned earlier about the sections not using all the width of the screen 

_Please be sure that your design occupies all the width of the mobile screen. It's probably an issue with hard-coded widths or some issues with your margins. That is causing an empty space on the right of the screen_

#### The issue 
![image](https://user-images.githubusercontent.com/15807343/149386299-fa402cde-d40c-4c8c-aa8a-4e0e44035c36.png)